### PR TITLE
Avatar refactor

### PR DESCRIPTION
**Pull request by matikrk**
* sizes -> SIZE
* iconTypes -> ICON_TYPE
* iconColors -> ICON_COLOR
* 1 component - 1 file
* avatar examples - profile icon

### CODE REVIEW

File: src/components/avatar/AvatarContent.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon, {types as ICON_TYPE, colors as ICON_COLOR} from '../icons/Icon';
+import {SIZE} from './avatarConstants';
+
+const AvatarContent = ({imgSrc, iconType, iconColor, iconSize}) => {
+  if (imgSrc) {
+    return <img className="sg-avatar__image" src={imgSrc}/>;
```
Comment (aju) [inline src/components/avatar/AvatarContent.jsx]: I don't know if its good idea to split it that much, its same component @kdzwinel @vergilius

File: src/components/avatar/AvatarContent.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Icon, {types as ICON_TYPE, colors as ICON_COLOR} from '../icons/Icon';
```
Comment (vergilius) [inline src/components/avatar/AvatarContent.jsx]: it's kind of weird / tricky to pass this around.
I mean, it's a part of icon module, but we're exporting it in two places (icon/avatar-content) with different identifiers. @aju @kdzwinel wdyt?

Comment (vergilius) [inline src/components/avatar/AvatarContent.jsx]: we're exporting it even in avatar module, again as ICON_TYPE

Comment (vergilius) [inline src/components/avatar/AvatarContent.jsx]: I mean, there should be single source of this, right?

Comment (aju) [inline src/components/avatar/AvatarContent.jsx]: I think it should be not reexported no point in that

File: src/components/avatar/AvatarContent.jsx (hunk 1 above)
Comment (vergilius) [inline src/components/avatar/AvatarContent.jsx]: If we don't use it separately, I'd suggest to merge it

Comment (matikrk) [inline src/components/avatar/AvatarContent.jsx]: IMHO splitting everywhere is no needed, but here it's gave more readability.

Comment (aju) [inline src/components/avatar/AvatarContent.jsx]: If we can't use something in separtion I think we should not split it

File: src/components/avatar/AvatarContent.jsx (hunk 2 above)
Comment (matikrk) [inline src/components/avatar/AvatarContent.jsx]: Will be more messy if you need import icon type from Icon module and then use it in avatar. Now you use avatar with dependencies exported from same file.

Comment (matikrk) [inline src/components/avatar/AvatarContent.jsx]: All imports in constants file

**Pull request merged**